docs/deployed-contracts/sepolia.md

```diff
@@ -180,3 +179,16 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
 - L2ERC20TokenBridge: [`0x7721F53d153Ae3CF937605fF1Bbb7D51B14E7902`](https://explorer.testnet.zircuit.com/address/0x7721F53d153Ae3CF937605fF1Bbb7D51B14E7902) (proxy)
 - L2ERC20TokenBridge: [`0x247f56cFc9021aeC161a4366412636ea33101D2B`](https://explorer.testnet.zircuit.com/address/0x247f56cFc9021aeC161a4366412636ea33101D2B) (impl)
 - Optimism Governance Bridge Executor: [`0x989CD486c02bfBe5c2D3C157cDCab099134e7697`](https://explorer.testnet.zircuit.com/address/0x989CD486c02bfBe5c2D3C157cDCab099134e7697)
+
+### Starknet
+
+##### Ethereum part
+
+- **(proposed)** wstETH StarknetERC20Bridge: [`0xaCbb1B9021eC68370B9821d5F36ddE26796bb436`](https://sepolia.etherscan.io/address/0xaCbb1B9021eC68370B9821d5F36ddE26796bb436) (proxy)
+- **(proposed)** wstETH StarknetERC20Bridge: [`0x0e8f4faBc96C2B16fa627F06229a77a765D8bFe0`](https://sepolia.etherscan.io/address/0x0e8f4faBc96C2B16fa627F06229a77a765D8bFe0) (impl)
```

Comment:
@TheDZhon any way this can be verified?

Reply:
You can learn more about these **testnet** deployments from the following:
- https://www.notion.so/StarkNet-bridge-testing-b91025c882e64e41980ec33b397d13c0?pvs=4#40a1e3a6a0054631bc18ec773e569c15 (internal QA report)
- Sepolia contracts: https://research.lido.fi/t/wsteth-deployment-on-starknet/6335/9 (externally published address)
- try it yourself here: https://sepolia.starkgate.starknet.io/